mysql-test/columnstore/basic/t/pron.test

```diff
@@ -0,0 +1,29 @@
+DROP DATABASE IF EXISTS pron;
```

Comment:
Please add `--disable_warnings` and `--enable_warnings` around the `DROP DATABASE IF EXISTS` statement here.

`--source ../include/have_columnstore.inc` is missing at the start of the test.

Also, please add a comment at the top explaining the test case.

Reply:
++